app/templates/privacy.html

```diff
@@ -0,0 +1,54 @@
+<style>
+#ppBody
+{
+    font-size:11pt;
+    width:100%;
+    margin:0 auto;
+    text-align:justify;
+}
+
+#ppHeader
+{
+    font-family:verdana;
+    font-size:21pt;
+    width:100%;
+    margin:0 auto;
+}
+
+.ppConsistencies
+{
+    display:none;
+}
+</style><div id='ppHeader'>downweek.com Privacy Policy</div><div id='ppBody'><div class='ppConsistencies'><div class='col-2'>
```

Comment:
please be consistent `<div id="ppHeader">` and the same for others in this line. Use double quotes for attributes values in html tags